cli/package.json

```diff
@@ -2,9 +2,15 @@
   "name": "immich-cli",
   "dependencies": {
     "axios": "^1.4.0",
+    "byte-size": "^8.1.1",
+    "chai": "^4.3.7",
```

Comment:
This is a testing library so it probably should not be here. If it doesn't compile there is probably another test file that needs to be excluded.

Reply:
Oh sorry, I missed that one, fixed! :D